bin/boom

```diff
@@ -106,10 +102,109 @@ def start_container(client, cmd, name, volumes=None, network_mode='host'):
 
         return None
 
+
+## Function to Upload the Zipped modules and input data file to S3 Bucket
+#  Create an EC2 instance, download files from S3
+#  Run the modules on the EC2 instance
+#  Upload the results back to S3
+#  @param tf The terraform object to create the resources
+#  @param tf_variables The terraform variables for resource creation
+#  @param infra_dir The directory path for the infra folder containing the
+# terraform code to create the resources
+def upload_object(tf, tf_variables, infra_dir):
```

Comment:
Please rename this function to `upload_and_run_object` or break down the functionalities into several functions.

Reply:
i can change the name. the functionalities cannot be broken down into smaller python code as the work is being done by terraform code and not python code.